45_梁鋆烨 第一课作业

RyanjunyeLIANG opened the pull request:
@hezhihua81

RyanjunyeLIANG committed:
test

test

RyanjunyeLIANG committed:
Delete test.html

RyanjunyeLIANG committed:
Update and rename yours.sol to Unit1salarycontract.sol

Unit 1 - A simple smart salary contract

RyanjunyeLIANG committed:
Update Unit1.sol and add comments

hezhihua81 reviewed (changes requested):
1. toEmployee 和 adjustSalary 在更改地址或工资前需要结清之前的工资（如果有的话），并重设lastPayDay。
2. 最后一个方法 getPaid 的格式排版有点问题。

RyanjunyeLIANG committed:
1st patch of  Unit1salarycontract.sol

Fixed problems of identification and pay the rest salary before adjust salary and employee's address

RyanjunyeLIANG committed:
minor bug fixed

hezhihua81 reviewed (commented):
一般比较地址写成和16进制0比较： employee != 0x0。和十进制0比较问题也不大
